### PR TITLE
Feat/122 and 121

### PR DESCRIPTION
**Pull request by amylo**
I branched from @mat10tng [model refactoring branch](https://github.com/factn/resilience-app/pull/412) because it has the updated model changes. 

We can merge this after `feat/model-consolidation` has merged into master.

### CODE REVIEW

**Comment by amylo** [comment]
Woops, I forgot to add the checkmark icon in the Accept Missions group button. Will do that now...

File: src/app/component/MissionGroup/MissionGroup.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    missions.forEach((mission) => {
+      action(mission.uid);
+    });
+  };
+
+  return (
+    <MuiExpansionPanel className="mission-group" defaultExpanded={true}>
+      <MuiExpansionPanelSummary
+        expandIcon={<MuiExpandMoreIcon className={classes.expandMoreIcon} />}
+        className={`mission-group-heading ${classes.expansionPanelSummary}`}
```
Comment (jwu910) [inline src/app/component/MissionGroup/MissionGroup.jsx:56]: We have [clsx](https://www.npmjs.com/package/clsx) a lightweight classnames library you can use to build classname strings if you want, don't seem to have too much complex classname generation here, thoughts?

Comment (amylo) [reply src/app/component/MissionGroup/MissionGroup.jsx:57]: Sure, i can use `clsx` if you want

**Comment by amylo** [comment]
**Volunteer Dashboard - Available** tab:
volunteer_dashboard-available

**Volunteer Dashboard - Accepted** tab:
volunteer_dashboard-accepted

**Volunteer Dashboard - Started** tab:
volunteer_dashboard-started

**Pull request merged**